app/assets/javascripts/application.js

```diff
@@ -10,8 +10,21 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
+//= require jquery3
 //= require rails-ujs
 //= require activestorage
 //= require turbolinks
 //= require trix
 //= require_tree .
+$(document).ready(function() {
```

Comment:
This needs to be `document.addEventListener('turbolinks:load', function() {` to make sure it works with Turbolinks.